<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>555</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>Parametrs</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>-100</y>
             <width>619</width>
             <height>411</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item>
             <layout class="QFormLayout" name="formLayout">
              <item row="1" column="0">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Output folder</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0" colspan="2">
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLineEdit" name="txtPathToProtocols">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="baseSize">
                   <size>
                    <width>16</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="toolButton_protocol">
                  <property name="text">
                   <string>...</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Roads</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0" colspan="2">
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QComboBox" name="cmbLayersRoad">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="baseSize">
                   <size>
                    <width>16</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>speed</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="cmbFieldsSpeed">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="baseSize">
                   <size>
                    <width>16</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>type road</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="cmbLayersRoad_type_road">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>direction</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="cmbFieldsDirection">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="baseSize">
                   <size>
                    <width>16</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="7" column="0">
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>Origins</string>
                </property>
               </widget>
              </item>
              <item row="8" column="0" colspan="2">
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QComboBox" name="cmbLayers">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="baseSize">
                   <size>
                    <width>16</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>id=</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="cmbLayers_fields">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="cbSelectedOnly1">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Selected features only</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="9" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Destinations</string>
                </property>
               </widget>
              </item>
              <item row="10" column="0" colspan="2">
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QComboBox" name="cmbLayersDest">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="baseSize">
                   <size>
                    <width>16</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>id=</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="cmbLayersDest_fields">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="cbSelectedOnly2">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Selected features only</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="12" column="0">
               <widget class="QLabel" name="lblVusual">
                <property name="text">
                 <string>Basic visualization layer</string>
                </property>
               </widget>
              </item>
              <item row="13" column="0" colspan="2">
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QComboBox" name="cmbVisLayers">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="baseSize">
                   <size>
                    <width>16</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>id=</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="cmbVisLayers_fields">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_12">
                  <property name="text">
                   <string>aliase</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txtAliase"/>
                </item>
               </layout>
              </item>
              <item row="14" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Default speed value (km/h)</string>
                </property>
               </widget>
              </item>
              <item row="14" column="1">
               <widget class="QLineEdit" name="txtSpeed">
                <property name="text">
                 <string/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight</set>
                </property>
               </widget>
              </item>
              <item row="15" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Maximal time travel, min</string>
                </property>
               </widget>
              </item>
              <item row="15" column="1">
               <widget class="QLineEdit" name="txtMaxTimeTravel">
                <property name="text">
                 <string/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight</set>
                </property>
               </widget>
              </item>
              <item row="18" column="0">
               <widget class="QCheckBox" name="cbUseFields">
                <property name="text">
                 <string>Run aggregate</string>
                </property>
               </widget>
              </item>
              <item row="19" column="0">
               <widget class="QLabel" name="lblFields">
                <property name="text">
                 <string>Field to aggregate</string>
                </property>
               </widget>
              </item>
              <item row="19" column="1">
               <widget class="QComboBox" name="cmbFields">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="baseSize">
                 <size>
                  <width>16</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="20" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Time bin for storing the results, min</string>
                </property>
               </widget>
              </item>
              <item row="20" column="1">
               <widget class="QLineEdit" name="txtTimeInterval">
                <property name="text">
                 <string/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Log</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTextBrowser" name="textLog"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QTextBrowser" name="textInfo"/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBreakOn">
       <property name="text">
        <string>Break</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="gridFrame">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lblMessages">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::NoButton</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>textInfo</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>btnBreakOn</tabstop>
  <tabstop>textLog</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
